config/routes/Payment_method_routes.js

```diff
@@ -0,0 +1,24 @@
+//  === Requirements ===
+var express = require('express')
+var Payment_method_routes = express.Router()
```

Comment:
File name, and all the Payment_method_routes need to be updated to lower_case please.

Reply:
All changed